docs/tutorial/tutorial-0.rst

```diff
@@ -77,6 +77,10 @@ button with have a margin of 50 pixels on each side. If we wanted to define a
 margin of 20 pixels on top of the button, we could have defined ``margin_top=20``,
 or we could have specified the ``margin=(20, 50, 50, 50)``.
 
+	button.style.flex = 1
+
+The flex property sets the flexible length on flexible items. If the element is not flexible, the flex property has no use. In this case, it gives a flexibility of 1 to the button created.
+
```

Comment:
Hi @starlord1311 thanks or the contribution

>In this case, it gives a flexibility of 1 to the button created.

This still sounds cryptic. Any chance of including a link with more information or actually explaining what having a flexibility of one means? (a simple example could do this!)